specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/restorePoint.json

```diff
@@ -713,24 +689,32 @@
           "items": {
             "$ref": "#/definitions/RestorePointSourceVMDataDisk"
           },
-          "x-ms-identifiers": [
-            "lun"
-          ],
+          "x-ms-identifiers": ["lun"],
           "description": "Gets the data disks of the VM captured at the time of the restore point creation."
+        },
+        "diskControllerType": {
```

Comment:
Pls. indicate that this property is read-only as the user cannot specify this property in the source metadata of restore point resource.